### PR TITLE
New Icon component

### PR DESCRIPTION
**Pull request by wEprous**
This Pull Request meets the following criteria:

* [x] Tests have been added/adjusted for my new feature
* [x] New Components are registered in index.js of my project

### CODE REVIEW

File: src/Icon/index.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import Icon from "react-icon-base";
+import * as tokens from "@kiwicom/orbit-design-tokens";
+
+export const iconSizes = {
+  small: tokens.widthIconSmall,
+  medium: tokens.widthIconMedium,
+  large: tokens.widthIconLarge,
+};
+
+export const iconColors = {
```
Comment (tomashapl) [inline src/Icon/index.js]: could be this defined in tokens please ?

File: src/Icon/index.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+// @flow
+import * as React from "react";
+import Icon from "react-icon-base";
+import * as tokens from "@kiwicom/orbit-design-tokens";
+
+export const iconSizes = {
```
Comment (tomashapl) [inline src/Icon/index.js]: define in tokens please

File: src/SystemMessage/index.js [hunk 3]
```diff
@@ -27,15 +27,15 @@ const SystemMessage = (props: Props) => (
     <Message type={props.type}>{props.children}</Message>
```
Comment (tomashapl) [inline src/SystemMessage/index.js]: looks broken: 
image

File: src/Icon/index.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  customColor: string,
+  children: React.Node,
+  viewBox: string,
+};
+
+const OrbitIcon = (props: Props) => {
+  const { size, color, customColor, children, viewBox } = props;
+  return (
+    <Icon
+      viewBox={viewBox}
```
Comment (tomashapl) [inline src/Icon/index.js]: There is no attribute viewBox in `react-icon-base` AFAIK

File: src/Icon/index.js (hunk 1 above)
Comment (tomashapl) [reply src/Icon/index.js]: Sorry i didn't noticed, it's ok ✅

File: src/Icon/index.js (hunk 2 above)
Comment (tomashapl) [reply src/Icon/index.js]: Sorry i didn't notice, it's ok ✅

File: src/SystemMessage/index.js (hunk 3 above)
Comment (wEprous) [reply src/SystemMessage/index.js]: Will fix it.

File: src/Icon/index.js (hunk 4 above)
Comment (wEprous) [reply src/Icon/index.js]: You are right, but in `react-icon-base` there is `...props` and we also need `viewBox` directly in SVG so we can resize it properly.

Comment (tomashapl) [reply src/Icon/index.js]: but built icon already has viewBox right ?

Comment (tomashapl) [reply src/Icon/index.js]: Solved. We need it for `react-icons-base` ¯\_(ツ)_/¯

File: config/build.js [hunk 5]
```diff
@@ -34,16 +34,20 @@ const svgo = new SVGO();
 names.forEach(async ({ inputFileName, outputComponentFileName, functionName }) => {
   const dom = await JSDOM.fromFile(inputFileName);
   const shapes = dom.window.document.querySelector("svg").innerHTML;
-  const svg = `<svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">${shapes}</svg>`;
+  const svg = `<svg viewBox="0 0 24 24">${shapes}</svg>`;
   const optimizationResult = await svgo.optimize(svg);
   const optimizedSvg = optimizationResult.data;
   const component = `/* eslint-disable */
     import * as React from "react";
+    import { default as Icon } from "../Icon";
```
Comment (tomashapl) [inline config/build.js]:
```
import OrbitIcon from "../Icon";
```

Comment (wEprous) [reply config/build.js]: Solved.

**Pull request merged**